### PR TITLE
BaSP-TG: Create, modify, delete, and get employee and filter list

### PR DESCRIPTION
**Pull request by lucasrendo**
Add the following features:
- Create an employee and write it's data in the database (JSON file)
- Find an employee from the database by it's id
- sort employees by name, last name or if they are active
- Delete an employee from the database
- Modify employee information

### CODE REVIEW

**Review by FiolAxel — APPROVED** [review]

File: src/resources/employees.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import express from 'express';
+import fs from 'fs';
+
+const employees = require('../data/employees.json');
+
+const router = express.Router();
+
+// Body parser
+router.use(express.json());
```
Comment (cocolobos) [inline src/resources/employees.js:9]: remove body parser, we have it on index.js

File: src/resources/employees.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        res.status(400).json(err);
+      } else {
+        res.json({ msg: 'Employee created', newEmployee });
+      }
+    });
+  }
+});
+
+// Edit an employee (no validations for wrong information)
+router.put('/id/:id', (req, res) => {
```
Comment (cocolobos) [inline src/resources/employees.js:111]: we can avoid using /id on the route maybe /:id is ok what do you think ?

File: src/resources/employees.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+};
+
+router.get('/', (req, res) => {
+  res.status(200).json({
+    data: employees,
+  });
+});
+
+// get 1 employee
+router.get('/id/:id', (req, res) => {
```
Comment (cocolobos) [inline src/resources/employees.js:31]: maybe we can avoid using /id with /:id is fine

Comment (lucasrendo) [reply src/resources/employees.js:31]: Same case as before

**Review by cocolobos — APPROVED** [review]

**Pull request merged**